x-pack/plugins/apm/public/components/app/correlations/latency_correlations.tsx

```diff
@@ -101,8 +142,11 @@ export function LatencyCorrelations({ onClose }: Props) {
       transactionName,
       transactionType,
       durationPercentile,
-      fieldNames,
+      // eslint-disable-next-line react-hooks/exhaustive-deps
+      fieldNames.join(','),
```

Comment:
Maybe you could extract it to a variable and reuse it here and on line 127?

Reply:
Thanks that's much better